lib/cell.rb

```diff
@@ -16,11 +20,25 @@ def place_ship(ship)
     @ship = ship
   end
 
+# We shouldn't be able to fire upon the same cell twice
```

Comment:
I think we will need to change this method so that we are not able to fire upon the same cell twice.